plugins/org.python.pydev.parser/src/org/python/pydev/parser/grammar_fstrings/build.xml

```diff
@@ -1,7 +1,10 @@
 <project name="jython-parser" default="parser">
+	
+	
 
-    <property name="javaccHome" value="c:\bin\javacc-5.0" />
-    <property name="python" value="c:\bin\Miniconda\envs\py27_tests\python.exe" />
+	<property name="env" value="C:\Users\olccjr"/>
```

Comment:
See: https://coderanch.com/t/108737/build-tools/read-environment-variables-Ant (or google for `ant use environment variable` for more info).